i18n/de.json

```diff
@@ -2947,10 +2947,10 @@
   "cloud.startTrial.modal.btn": "Starte Test",
   "cloud_delinquency.banner.buttonText": "Rechnung jetzt aktualisieren",
   "cloud_delinquency.banner.title": "Aktualisieren Sie jetzt Ihre Rechnungsdaten, um bezahlte Funktionen zu behalten.",
-  "cloud_free.professional_feature.back": "Zurück",
```

Comment:
this PR is having merge conflicts, do you need to actually update i18n files besides the en.json?

Reply:
I just updated the translation keys by removing `cloud` at the beginning since the prompt is now for both cloud and self-managed